src/Distributions.jl

```diff
@@ -207,6 +209,7 @@ export
     sqmahal,            # squared Mahalanobis distance to Gaussian center
     sqmahal!,           # inplace evaluation of sqmahal
     location,           # get the location parameter
+    location!,          # provide storage for the location parameter (used in multivariate distribution mvlognormal)
```

Comment:
I'm a little hesitant to export this without support for more distributions, but it does follow the pattern of many other functions defined here.  Same for `params!` and `scale!` below.

Can you open an issue for defining these for other distributions?